modules/prebidServerBidAdapter/index.js

```diff
@@ -717,18 +720,28 @@ Object.assign(ORTB2.prototype, {
               min.ref = min.min = floor;
             } else {
               const value = convertCurrency(floor.floor, floor.currency, min.ref.currency);
+
               if (value != null && value < min.ref.floor) {
                 min.ref.floor = value;
                 min.min = floor;
               }
             }
+
             return min;
           }, {}).min
       })();
 
       if (floor) {
         imp.bidfloor = floor.floor;
         imp.bidfloorcur = floor.currency;
+
+        if (floorMinCur == null) { floorMinCur = floor.currency }
+        const convertedFloorValue = convertCurrency(floor.floor, floor.currency, floorMinCur);
+        const ortb2ImpFloorMin = imp.ext?.prebid?.floors?.floorMin || imp.ext?.prebid?.floorMin;
+        const lowestImpFloorMin = ortb2ImpFloorMin && ortb2ImpFloorMin < convertedFloorValue ? ortb2ImpFloorMin : convertedFloorValue;
+
+        deepSetValue(imp, 'ext.prebid.floors.floorMin', lowestImpFloorMin);
```

Comment:
Here's a test case: you have a bidRequest with 
```
  getFloor() {return {currency: 'JPY', floor: 2}},
  ortb2Imp: {
     ext: { prebid: { floors: { floorMinCur: 'USD', floorMin: 1 }}}
  }
```

Since 2 JPY is less than 1 USD, the expectation is that `imp[].ext.prebid.floors` is set to either 2 JPY, or its equivalent in USD. I think this implementation would set 1 JPY.